data/com.github.finefindus.eyedropper.metainfo.xml.in.in

```diff
@@ -49,6 +49,7 @@
   <releases>
     <release version="2.1.0" date="2025-03-23">
       <description translatable="no">
+        <p>In this version:</p>
```

Comment:
This isn't a super great string, as Flathub/GNOME Software already display a similar string above the release notes
![Flathub release notes](https://github.com/user-attachments/assets/974cfde4-5418-4280-8c57-bc2c01bfc88c)